microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -55,10 +67,55 @@ def check_dpdk_build(self, node: Node, log: Logger) -> None:
             f"TX-PPS ({tx_pps}) should have been greater than 2^20 (~1m) PPS."
         ).is_greater_than(2 ** 20)
 
-    def _execute_expect_zero(self, node: Node, cmd: str) -> str:
-        result = node.execute(cmd, sudo=True, shell=True)
-        result.assert_exit_code()
-        return result.stdout
+    @TestCaseMetadata(
+        description="""
+            Tests a basic sender/receiver setup.
+        """,
+        priority=2,
+        requirement=simple_requirement(
+            min_nic_count=2,
+            network_interface=Sriov,
+            min_count=2,
+        ),
+    )
+    def check_2node_snd_rcv(self, environment: Environment, log: Logger) -> None:
+        test_kits = _init_nodes_concurrent(environment, log)
+        sender, receiver = test_kits
+
+        # helpful to have the public ip for debugging
+        external_ips = [
+            x.node.execute("curl ifconfig.me").stdout for x in [sender, receiver]
+        ]
+        log.debug((f"\nsender:{external_ips[0]}\nreceiver:{external_ips[1]}\n"))
+
+        snd_nic, rcv_nic = [x.node_nic_info.get_nic("eth1") for x in test_kits]
```

Comment:
It looks hard code to the eth1. What's the reason to add like this way? Add comments at least.

Reply:
Oop, no reason. Will fix.